File: solutions/c/isogram/1/isogram.c
#include "isogram.h"
#include <string.h>

bool is_isogram(const char phrase[])
{
    if (phrase == 0)
        return (false);
    for (size_t i = 0; i<strlen(phrase); i++)
    {
        for (size_t j = 0; j<strlen(phrase); j++)
        {
            if (((phrase[i] >= 'A' && phrase[i] <= 'Z') || (phrase[i] >= 'a' && phrase[i] <= 'z')) && i != j && (phrase[i] == phrase[j] || phrase[i] == phrase[j] + 32 || phrase[i] + 32 == phrase[j]))
                return (false);
        }
    }
    return (true);
}

File: solutions/csharp/isogram/1/Isogram.cs
using System;

public static class Isogram
{
    public static bool IsIsogram(string word)
    {
        for (int i = 0; i < word.Length - 1; i++)
        {
            for (int j = i + 1; j < word.Length; j++)
            {
                if (Char.IsLetter(word[i]) && Char.ToLower(word[i]) == Char.ToLower(word[j]))
                    return false;
            }
        }
        return true;
    }
}
